x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/use_bar_charts.ts

```diff
@@ -19,17 +19,27 @@ export const useBarCharts = ({ data = [] }: UseBarHookProps) => {
   useEffect(() => {
```

Comment:
I don't believe it was discussed. I think a lot about the waterfall could be organized better from a file perspective. I'd like to bring it up to address at another time.

Reply:
i think for a generic hook it makes sense to move it to a hooks dir, but for a hook like this, it totally make sense to keep it local. since it's not being used anywhere.